MerlinAU.sh

```diff
@@ -4073,8 +4075,8 @@ _EntwareServicesHandler_()
    if [ -f /opt/bin/diversion ]
    then
        Say "${actionStr} Diversion service..."
-       /opt/bin/diversion "$divAction"
-       sleep 1
+       /opt/bin/diversion "$divAction" &
```

Comment:
I like the idea of running this in the background.

Reply:
> I like the idea of running this in the background.
>
My thinking was to try to avoid any possible interruption to the F/W Update.